src/services/formatting/formatting.ts

```diff
@@ -1117,6 +1117,25 @@ namespace ts.formatting {
         }
     }
 
+    export function getRangeOfEnclosingComment(sourceFile: SourceFile, position: number, onlyMultiLine: boolean): CommentRange | undefined {
+        const precedingToken = findPrecedingToken(position, sourceFile);
+        const trailingRangesOfPreviousToken = precedingToken && getTrailingCommentRanges(sourceFile.text, precedingToken.end);

```

Comment:
If this were set to empty, rather than undefined, in the absence of `precedingToken`, the checks below could be simplified.